### PR TITLE
Added option to set interface MTU for containers

### PR DESCRIPTION
**Pull request by mandeepdhami**
Added option to set interface MTU for containers
Added basic tests for it
Bumped version number

### CODE REVIEW

File: provision/acc_provision/templates/aci-containers.yaml [hunk 1]
```diff
@@ -71,6 +71,9 @@ data:
         "service-vlan": {{ config.net_config.service_vlan|json }},
         "encap-type": {{ config.node_config.encap_type|json }},
         "aci-infra-vlan": {{ config.net_config.infra_vlan|json }},
+        {% if config.net_config.interface_mtu %}
+        "interface-mtu": {{ config.net_config.interface_mtu }},
```
Comment (snaiksat) [inline provision/acc_provision/templates/aci-containers.yaml]: Does this need the '|json" filter as is being used for other configs?

File: provision/acc_provision/acc_provision.py [hunk 2]
```diff
@@ -545,6 +546,22 @@ def config_adjust(args, config, prov_apic, no_random):
     return adj_config
 
 
+def is_valid_mtu(xval):
+    if xval is None:
+        # use default configured on thsi host
```
Comment (snaiksat) [inline provision/acc_provision/acc_provision.py]: nit: thsi -> this

File: provision/acc_provision/acc_provision.py [hunk 3]
```diff
@@ -545,6 +546,22 @@ def config_adjust(args, config, prov_apic, no_random):
...
+        return True
+
+    xmin = 1280   # for IPv6
```
Comment (snaiksat) [inline provision/acc_provision/acc_provision.py]: Seems like the default we were are using is 1500:
https://github.com/noironetworks/aci-containers/blob/f1a564c9abdf320eaf3af3654bbd5c02c092ce96/pkg/hostagent/config.go#L177

Should xmin be 1500?

File: provision/acc_provision/templates/aci-containers.yaml (hunk 1 above)
Comment (mandeepdhami) [reply provision/acc_provision/templates/aci-containers.yaml]: Good catch, I will fix that.

File: provision/acc_provision/acc_provision.py (hunk 2 above)
Comment (mandeepdhami) [reply provision/acc_provision/acc_provision.py]: OK ;-)

File: provision/acc_provision/acc_provision.py (hunk 3 above)
Comment (mandeepdhami) [reply provision/acc_provision/acc_provision.py]: No, the user can set it below the default - just not lower than 1280 as IPv6 requires that (IPv4 min is lower, but opflex supports IPv6 so we will go with that min).

**Pull request merged**